src/main/java/org/apache/commons/text/similarity/package-info.java

```diff
@@ -29,6 +29,7 @@
  * <li>{@link org.apache.commons.text.similarity.FuzzyScore Fuzzy Score}</li>
  * <li>{@link org.apache.commons.text.similarity.HammingDistance Hamming Distance}</li>
  * <li>{@link org.apache.commons.text.similarity.JaroWinklerDistance Jaro-Winkler Distance}</li>
+ * <li>{@link org.apache.commons.text.similarity.JaroWinklerSimilarity Jaro-Winkler Similarity}</li>
```

Comment:
Thanks for updating the docs!!! :clap: Would you mind updating `userguide.xml` too?

Reply:
fixed